filament/backend/CMakeLists.txt

```diff
@@ -575,6 +575,12 @@ if (APPLE OR LINUX)
         test/test_CircularBuffer.cpp
         test/test_CommandBufferQueue.cpp
     )
+    if (APPLE)
+        # Metal-specific tests
+        list(APPEND BACKEND_TEST_SRC
+            test/test_MetalBlitter.mm
```

Comment:
nit: maybe this can be `test/metal/test_MetalBlitter.mm` 
(parallel to `backend/src/metal`)